jvcl/run/JvAni.pas

```diff
@@ -41,6 +41,9 @@ interface
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
   Classes, RTLConsts, Windows, Graphics, Controls, ExtCtrls, Dialogs,
+  {$IFDEF HAS_UNIT_SYSTEM_UITYPES}
+  UITypes,
```

Comment:
Same remark as on other pull request, use `System.UITypes`, it's the real name of the unit